doc/API.md

```diff
@@ -368,6 +369,38 @@ disregarded for forward compatibility.
 None; the `result` field will be set to the empty object `{}`. Any value should be
 disregarded for forward compatibility.
 
+### `gethistory`
+
+`gethistory` retrieves a paginated list of accounting events.
+
+The new deposit created by a change output is not included in the list
+of events.
+
+The `amount` of a `spend` event resource does not include the value of
+the change output.
```

Comment:
I wonder how we'll be able to rule out deposits that are a change..

Also, i'm not fond of special casing specific change outputs. How about:
> Aiming at giving an accounting point of view, the amounts returned by this call are the total of inflows and outflows net of any change amount (that is technically a transaction output, but not a cash outflow).

Reply:
Then we should maybe add the txid in the event resource, so the client can find change output amount itself

Reply:
I don't understand